bin/ci-annotate-errors

```diff
@@ -12,7 +12,8 @@
 # ci-annotate-errors - Detect errors in junit xml as well as log files during
 # CI and find associated open GitHub issues in Materialize repository.
 
-HOSTNAME=$(exec "$(dirname "$0")"/ci-builder run stable "$(dirname "$0")"/mzcompose --mz-quiet --find get-cloud-hostname run default --app-password-env-var "PRODUCTION_ANALYTICS_APP_PASSWORD")
+# Only take last two lines, previous content can be retries when docker pull has problems
+HOSTNAME=$(exec "$(dirname "$0")"/ci-builder run stable "$(dirname "$0")"/mzcompose --mz-quiet --find get-cloud-hostname run default --app-password-env-var "PRODUCTION_ANALYTICS_APP_PASSWORD" | tail -n2)
```

Comment:
Doesn't this contain a newline then? How can the hostname have a newline?
```suggestion
HOSTNAME=$(exec "$(dirname "$0")"/ci-builder run stable "$(dirname "$0")"/mzcompose --mz-quiet --find get-cloud-hostname run default --app-password-env-var "PRODUCTION_ANALYTICS_APP_PASSWORD" | tail -n2 | head -n1)
```

Reply:
This is done in line 18 already...

Reply:
I slightly refactored it now.